xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -944,10 +947,11 @@ static void DisplayReconfigured(CGDirectDisplayID display,
   int h;
   double fps;
 
-  int dispIdx = GetDisplayIndex(CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(
-      CSettings::SETTING_VIDEOSCREEN_MONITOR));
+  const NSUInteger dispIdx =
+      GetDisplayIndex(CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(
+          CSettings::SETTING_VIDEOSCREEN_MONITOR));
   GetScreenResolution(&w, &h, &fps, dispIdx);
-  NSString* dispName = screenNameForDisplay(GetDisplayID(dispIdx));
+  const NSString* dispName = screenNameForDisplay(dispIdx);
```

Comment:
`const NSString` doesn't make any sense because `NSString` already denotes an immutable string. If you want to make sure that `dispName` isn't reassigned, use `NSString* const` type.